mimic/rest/loadbalancer_api.py

```diff
@@ -26,20 +27,20 @@
 @implementer(IAPIMock, IPlugin)
 class LoadBalancerApi(object):
     """
-    Rest endpoints for mocked Load balancer api.
+    This class registers the load balancer API in the service catalog.
+    It does NOT implement individual endpoints.
```

Comment:
Personally, I would just remove this line, but if you feel strongly we can leave it for now. Regardless we probably need some kind of document laying out plugin architecture, a more formal version of @cyli's very helpful diagram wiki page.